apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -28,40 +38,61 @@ const DepartureList = ({
   );
 
   let departures: DepartureInfo[] = [];
+  const groupedAlerts = alertsByRoute(alerts);
+  const alertsForRoute = groupedAlerts[route.id] || [];
 
+  const alertsByDirectionObj = alertsByDirectionId(alertsForRoute);
+  const alertsAffectingBothDirectionsArray = alertsAffectingBothDirections(
+    alertsForRoute
+  );
+
+  const alertsDirectionArray = alertsByDirectionObj[directionId]
+    ? alertsByDirectionObj[directionId]
+    : [];
+
+  const allAlerts = concat(
+    alertsAffectingBothDirectionsArray,
+    alertsDirectionArray
+  ).filter(alert => {
+    return ["detour", "suspension", "shuttle"].includes(alert.effect);
+  });
```

Comment:
question: The acceptance criteria of this ticket distinguishes between some differences in alerts that should be displayed, for example: 

> * If there is a shuttle alert associated with the stop that is currently active or upcoming, it should be shown on the realtime tracking page.
> * If there is a shuttle alert elsewhere along the route that is currently active, it should be shown on the realtime tracking page.

Does this list of alerts account for this active or upcoming distinction for route-specific alerts?
Will the stop-level alerts be addressed as a separate PR?

Reply:
I believe the stop level alerts are already captured in `allAlerts`. If a stop is  closed wouldn't that also show up on the route as a detour or shuttle? Oh there is no distinction atm.